must-call-checker/src/main/java/org/checkerframework/checker/mustcall/MustCallTransfer.java

```diff
@@ -289,6 +329,13 @@ public void updateStoreWithTempVar(TransferResult<CFValue, CFStore> result, Node
     }
   }
 
+  /**
+   * Either returns the temporary variable associated with node, or creates one if one does not
+   * exist. node must be an expression, not a statement.
+   *
+   * @param node a node
+   * @return a temporary variable node representing node that can be placed into a store
```

Comment:
This would be clearer if the second `node` were `{@code node}`.  As written, the two occurrences of "node" that mean different things are confusing.